iphone/Classes/TiUILabel.m

```diff
@@ -448,6 +448,7 @@ - (void)setFont_:(id)font
     CGFloat ratio = minFontSize / label.font.pointSize;
     [label setMinimumScaleFactor:ratio];
   }
+  [self padLabel];
```

Comment:
This is a pretty major change. Can we validate the following use-cases for this:
- Multi-line labels
- ellipsize labels
- minimum-font-sizes

Reply:
Right, changing other label properties causes this issue too. Our label code probably needs to re-udpate/invalidate itself after a property change.
https://jira.appcelerator.org/browse/TIMOB-24864